src/components/classBased/TodosList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TodoItem from './TodoItem';
+
+const TodosList = (props) => {
+  const {
+    todos, handleChangeProps, deleteTodoProps, setUpdate,
+  } = props;
+
+  return (
+    <ul
+      className="w-full max-h-80 overflow-y-auto lg:w-11/12 flex flex-col align-center self-center mx-auto"
+    >
+      {todos.map((todo) => (
+        <TodoItem
+          key={todo.id}
+          todo={todo}
+          handleChangeProps={handleChangeProps}
+          deleteTodoProps={deleteTodoProps}
+          setUpdate={setUpdate}
+        />
+      ))}
+    </ul>
+  );
```

Comment:
The issues I am getting are related to these lines of code, these are the errors I am getting:
```
Failed prop type: TodosList: prop type `todos` is invalid; it must be a function, usually from the `prop-types` package, but received `undefined`.This often happens because of typos such as `PropTypes.function` instead of `PropTypes.func`.
```
```
Uncaught TypeError: Cannot read properties of null (reading 'map')
    at TodosList (TodosList.js:11:1)
```

Reply:
Locally it's working but I am still getting this error.
This prop is of type `Array` but I will try to understand why it's says it should be a function

Reply:
@jssol I don't think the issue is with the proptype, I tried printing the result of todos from TodoContainer and I got null too so maybe try checking from there, I hope this helps you.

Reply:
@mirouhml, I have fixed the prop-types checking.
Now it should be working fine